app/v5/bfLosses/list/model/LossType.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v5.bfLosses.list.model
+
+import play.api.libs.json._
+import utils.enums.Enums
+
+sealed trait LossType {
```

Comment:
This cuts across multiple endpoints. Consider making it, `TypeOfLoss`, and `IncomeSourceType` common code as I think, there is less risk of it getting out of sync.

If there is variation, it will be between schemas, not between endpoints so there is an argument for making this schema specific (not just endpoint specific). The problem, though, is that we'd have to maintain even more conversion methods (toLossType/toTypeOfLoss) that will be unused depending on the endpoint and so on the direction that the data is moving. Feels like one we should address when we need to, as I'm not sure this is the right approach,

Reply:
Agreed, they shouldn't be endpoint specific. Schema specific would make sense too- for now keeping them version specific might be better?

Reply:
Yes. Version specific for now would be good.

Reply:
Should these be in `v5.bfLosses.common.model`? Then other _common_ things can go in there (I think this has been done in other APIs). I am thinking things like the `ResolveLossId` that is still part of the def1.